lib/internal/process/execution.js

```diff
@@ -65,6 +65,9 @@ function evalScript(name, body, breakFirstLine, print) {
 
   // Create wrapper for cache entry
   const script = `
+    globalThis.node = new Proxy(Object.freeze(Object.create(null)), {
+      get: (_, prop) => require("node:"+prop),
```

Comment:
```suggestion
      get: (_, prop) => require('node:' + prop),
```
Code style consistency concerns.